src/components/button/__tests__/button.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react'
+
+import { Button } from "../button"
```

Comment:
use `@` everywhere for consistency

Reply:
- As it is just one folder back `@` feels like overkill to me as you are going back to the `src` folder and then importing the file which was just a folder back.